doc/user/content/sql/system-catalog.md

```diff
@@ -285,17 +286,33 @@ For per-worker frontier information, see
 
 Field       | Type       | Meaning
 ------------|------------|--------
-`global_id` | [`text`]   | The ID of the index that created the dataflow. Corresponds to [`mz_indexes.id`](#mz_indexes).
+`dataflow ` | [`text`]   | The ID of the index or materialized view that created the dataflow. Corresponds to [`mz_materializations.dataflow`](#mz_materializations).
```

Comment:
I think this change might be wrong. Reading up in the text (unchanged), it looks like the description here is not as precise as it could be. As far as I understand, this field will be the global id of the item that created the dataflow (index or materialized view), neither of which are a dataflow (there could be multiple of them in the dataflow). I think we want this to be a primary key and remain the global id of the item.

Reply:
The nomenclature is the same used in `mz_materialization_dependencies` currently, but it does not match the notion of dataflow ids or names from other catalog relations (e.g., `mz_dataflow_names`). Moritz suggests that we defer proper discussion of the renaming and documentation changes not related to the addition of `mz_materialization_source_frontiers` or `mz_worker_materialization_source_frontiers` to the system table revamp ticket by Nikhil: https://github.com/MaterializeInc/materialize/issues/7174. I will revisit this tomorrow and keep to `global_id` for now in the affected catalog relations.

Reply:
I made a second pass changing the referred attribute name in the sources/view that I added as well as `mz_materializations` to `global_id` for now. If this looks OK for now, then after merging this PR, I will add a note to the system revamp ticket to capture that this might not be the perfect attribute name and also that the nomenclature in `mz_materialization_dependencies` looks wrong.